synthtool/gcp/templates/node_library/api-extractor.json

```diff
@@ -0,0 +1,369 @@
+/**
+ * Config file for API Extractor.  For more info, please visit: https://api-extractor.com
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
+  /**

```

Comment:
should this be a `.js` file, seems to be very angry about the  comments in the JSON.

Reply:
It's supposed to be a `json` file, generated by api-extactor. I added a `.gitattributes` file instead so it looks less angry.